### ISSUE TITLE
Un-prefixed css rules in styles.css causing global issues in Moodle themes

### ISSUE THREAD

**Issue by cstones** [marcusgreen/moodle-qtype_gapfill#39]
Hi again, sorry to bother but the style rules that do not have a plugin specific prefix (ie. gap or gapfill_) so they are modifying the global Moodle styles in our sites and causing issues.

Could you add something gapfill related to the following when they appear?
I believe there is normally some kind of  body tag on pages that show question types
that might be useful. But I forget. 

```
.hascorrect {
    background: lightblue;
    border: solid thin black;
}

.hasnotcorrect {
    border: solid thin black;
    box-shadow: 3px 3px #7bc1f7;
    font-style: italic;
}

.attobackground {
    background: lightgrey;
    border-radius: .25rem;
    border: 1px solid rgba(0, 0, 0, .15);
    color: #55595c;
    display: block;
    height: 100%;
    left: 0;
    line-height: 1.25;
    padding: .5rem .75rem;
    position: absolute;
    top: 0;
    width: 100%;
}

.item {
    background: white;
}
```

**Comment by marcusgreen** [comment]
No bother whatsoever, that is an excellent catch. Those classes are appearing within the editor so the .que.gapfill doesn't apply. I have just done a commit that adds a gapfill prefix, i.e. .gapfill .hascorrect. etc. etc. Seems to work. If you have any end users with feature requests I would be interested. It would probably take forever to implement anything, but it is good to know what people like.

**Comment by marcusgreen** [comment]
Have just released v1.975 to the Moodle.org plugins database that includes those css prefixes. I believe this addresses this issue so I am closing it. If I have missed anything feel free to re-open/create another issue.
